src/vg/civcraft/mc/civchat2/CivChat2Manager.java

```diff
@@ -296,11 +299,12 @@ public void broadcastMessage(Player sender, String chatMessage, Set<Player> reci
 						//reciever is in differnt world dont send
 						continue;
 					} else {
-						receiver.sendMessage(sb.append(color) 
+						receiver.sendMessage(String.format(event.getFormat(), sender.getName(), chatMessage));
```

Comment:
Additionally it would be good to do a simple test -- if event.getFormat().equals( "MC default pattern" ) -- then replace with Civ-pattern.